site/content/how-to/traffic-management/routing-traffic-to-your-app.md

```diff
@@ -0,0 +1,371 @@
+
+Using these resources we will configure a simple routing rule to match all HTTP traffic with the hostname "cafe.example.com" and route it to the **coffee** service.
+
+## Set up
+
+Create the **coffee** application in Kubernetes by copying and pasting the following block into your terminal:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coffee
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: coffee
+  template:
+    metadata:
+      labels:
+        app: coffee
+    spec:
+      containers:
+      - name: coffee
+        image: nginxdemos/nginx-hello:plain-text
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: coffee
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: coffee
+EOF
+```
+
+This will create the **coffee** service and a deployment with two pods. Run the following command to verify the resources were created:
+
+```shell
+kubectl get pods,svc
+```
+
+Your output should include two **coffee** pods and the **coffee** service:
+
+```text
+NAME                          READY   STATUS      RESTARTS   AGE
+pod/coffee-7dd75bc79b-cqvb7   1/1     Running     0          77s
+pod/coffee-7dd75bc79b-dett3   1/1     Running     0          77s
+
+
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+service/coffee       ClusterIP   198.51.100.1     <none>        80/TCP    77s
+```
+
+## Application architecture with NGINX Gateway Fabric
+
+To route traffic to the **coffee** application, we will create a gateway and HTTPRoute. The following diagram shows the configuration we are creating in the next step:
+
+{{<img src="/img/route-all-traffic-config.png" alt="Configuration">}}
+
+We need a gateway to create an entry point for HTTP traffic coming into the cluster. The **cafe** gateway we are going to create will open an entry point to the cluster on port 80 for HTTP traffic.
+
+To route HTTP traffic from the gateway to the **coffee** service, we need to create an HTTPRoute named **coffee** and attach it to the gateway. This HTTPRoute will have a single routing rule that routes all traffic to the hostname "cafe.example.com" from the gateway to the **coffee** service.
+
+Once NGINX Gateway Fabric processes the **cafe** gateway and **coffee** HTTPRoute, it will configure its data plane (NGINX) to route all HTTP requests sent to "cafe.example.com" to the pods that the **coffee** service targets:
+
+{{<img src="/img/route-all-traffic-flow.png" alt="Traffic Flow">}}
+
+The **coffee** service is omitted from the diagram above because the NGINX Gateway Fabric routes directly to the pods that the **coffee** service targets.
+
+{{< note >}}In the diagrams above, all resources that are the responsibility of the cluster operator are shown in green. The orange resources are the responsibility of the application developers. 
```

Comment:
Approx 10% of males are red/green colorblind. Will these green resources make sense to those users if they can't see green?

Reply:
Good point. I will redo the image to avoid this and rewrite the note to avoid the red-green dependancy